var/spack/repos/builtin/packages/ibm-java/package.py

```diff
@@ -28,6 +28,7 @@ class IbmJava(Package):
     provides('java@8')
 
     conflicts('target=x86_64:', msg='ibm-java is only available for ppc64 and ppc64le')
+    conflicts('target=aarch64', msg='ibm-java is only available for ppc64 and ppc64le')
```

Comment:
```suggestion
    conflicts('target=aarch64:', msg='ibm-java is only available for ppc64 and ppc64le')
```
@alalazo is this right?